fileWithPRComments.js

```diff
@@ -0,0 +1,8 @@
+/* global QamineUtils */
+
+"use strict";
+
+(function($, window) {
```

Comment:
Codacy found an issue: [Move the invocation into the parens that contain the function.](https://app.dev.codacy.org/gh/devcodacytestuser/CheckPrCommentsPublicProjectDev/pullRequest?prid=23255)